Spanner/src/Connection/Grpc.php

```diff
@@ -152,6 +164,15 @@ public function __construct(array $config = [])
                 ? $config['authHttpHandler']
                 : null
         );
+
+        if (isset($config['enable-gcp-optimizer'])) {
```

Comment:
Our convention is to use camelCase for configuration options. `$config['enableGcpOptimizer']`.

Reply:
Done.